Classes/Form/AbstractInlineFormField.php

```diff
@@ -423,21 +423,22 @@ public function getLevelLinksPosition()
     }
 
     /**
-     * @param string $foreignSelectorFieldTcaOverride
+     * @param string $overrideChildTca
      * @return RelationFieldInterface
+     * @deprecated
      */
-    public function setForeignSelectorFieldTcaOverride($foreignSelectorFieldTcaOverride)
+    public function setOverrideChildTca($overrideChildTca)
     {
-        $this->foreignSelectorFieldTcaOverride = $foreignSelectorFieldTcaOverride;
+        $this->overrideChildTca = $overrideChildTca;
         return $this;
     }
 
     /**
      * @return string
      */
-    public function getForeignSelectorFieldTcaOverride()
+    public function getOverrideChildTca()
```

Comment:
Old method must remain in place to avoid a breaking change. Old method can call new method transparently.

Reply:
I added a getter and setter that replicates the overridChildTca structure.